io.c

```diff
@@ -10473,9 +10473,6 @@ static VALUE argf_readlines(int, VALUE *, VALUE);
  *    $cat t.txt | ruby -e "p readlines 12"
  *    ["First line\n", "Second line\n", "\n", "Fourth line\n", "Fifth line\n"]
  *
- *  With arguments +sep+ and +limit+ given, combines the two behaviors;
- *  see {Line Separator and Line Limit}[rdoc-ref:IO@Line+Separator+and+Line+Limit].
```

Comment:
Why was this removed?

Reply:
Multiple similar passages removed in previous PRs.  In my view, not needed;  plus, the cited section no longer exists (just the two separate ones).

Reply:
Why do some methods have a section about `sep` and `limit` and some methods don't?

Reply:
Probably an error.  Let's get the link fixes merged, then deal with that?

Reply:
I think this one should be fixed before it can be merged.

Reply:
Not sure what the fix should be.

Reply:
Well after this change, it seems like arguments `sep` and `limit` are mutually exclusive (especially with the wording "With only [...] argument [...] given"). The documentation needs to be obvious that the two can be combined and should be clear what the behaviour will be.

Reply:
Done (I think).